assembl/static2/js/app/components/debate/brightMirror/sideComment/sideCommentMenu.jsx

```diff
@@ -0,0 +1,139 @@
+// @flow
```

Comment:
~copy of harvestingMenu.jsx

Reply:
not exactly but it can be done later if we wish to apply the changes I did for this module to the harvesting functionality